web/routes/admin/personal-information.js

```diff
@@ -0,0 +1,31 @@
+const router = require('express').Router()
+const axios = require('axios')
+const url = require('../../settings.js')
+const middleware = require('../middleware/middleware.js')
+const role = 'ADMIN'
+
+router.get('/gegevens', middleware.authRole(role), function (req, res) {
+    const get_info = `${url}/users/userinfo/`
+    const token = req.signedCookies.Authorization
+
+    axios
+        .get(get_info, {
+            headers: {
+                Authorization: `${token}`,
+            },
+        })
+        .then((response) => {
+            const isNullField = Object.values(response.data).some(
+                (value) => value === null
+            )
+            res.render(
+                'admin/personal-information',
+                Object.assign({}, response.data, {
+                    isNullField: isNullField,
+                })
+            )
+        })
+        .catch((error) => console.log('error: ', error))
```

Comment:
```suggestion
        .catch((error) => console.log('error: ', error))
```
error.response.data hoort het te zijn